src/components/Channel/Channel.tsx

```diff
@@ -1011,7 +1013,7 @@ const ChannelInner = <
       BaseImage: props.BaseImage,
       CooldownTimer: props.CooldownTimer,
       CustomMessageActionsList: props.CustomMessageActionsList,
-      DateSeparator: props.DateSeparator,
+      DateSeparator: props.DateSeparator || DateSeparator,
```

Comment:
This default for `DateSeparator` and `MessageSystem` components was previously set up in `useMessageListElements`: https://github.com/GetStream/stream-chat-react/blob/master/src/components/MessageList/hooks/MessageList/useMessageListElements.tsx#L59-L63

I think it makes sense to set default in `Channel`, since it's where all other defaults are provided (e.g. `Message` defaults to `MessageSimple` here).